### PR TITLE
Added authors and added endpoint 

### LINKED ISSUE #1: Add a line with our names to the server
We want to add our names to the page.

### PR DESCRIPTION
**Pull request by yonikesel1**
added Yoni and Laurence to the root message.
Also added endpoint to return most liquid pair from SushiSwap.
resolves #1

### CODE REVIEW

**Comment by lmilbaum** [comment]
Please verify that the PR has an assignee and at least one reviewer who approved it before it is merged to main.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+node_modules
+.vscode
```
Comment (lmilbaum) [inline .gitignore:2]: end of line is missing

Comment (yonikesel1) [reply .gitignore:2]: Should I create a new PR for this change?

Comment (lmilbaum) [reply .gitignore:2]: yes :-)

**Pull request merged**